bin/copyUserCommentUpvoteData.ts

```diff
@@ -0,0 +1,41 @@
+import '../src/config';
+import createOrGetConnection from '../src/db';
+
+(async (): Promise<void> => {
+  const limitArgument = process.argv[2];
+  const offsetArgument = process.argv[3];
+
+  if (!limitArgument || !offsetArgument) {
+    throw new Error('limit and offset arguments are required');
+  }
+
+  const limit = +limitArgument;
+
+  if (Number.isNaN(limit)) {
+    throw new Error('limit argument is invalid, it should be a number');
+  }
+
+  const offset = +offsetArgument;
+
+  if (Number.isNaN(offset)) {
+    throw new Error('offset argument is invalid, it should be a number');
+  }
+
+  const con = await createOrGetConnection();
+
+  await con.transaction(async (manager) => {
+    await manager.query(`SET session_replication_role = replica;`);
```

Comment:
Disable triggers for session, to prevent `comment.upvote` being incremented another time since they were already incremented when rows were added to old `comment_upvote`.

Reply:
We need to make sure that:
* It actually works 😓 
* Doesn't have any implication on production queries
I think it should impact only the current transaction and work properly but just double check

Reply:
Yep, tested locally.